Restfull API/api/app/__init__.py

```diff
@@ -0,0 +1,19 @@
+from flask import Flask, Markup
+#local imports
+from config import app_config
+# Initialize the app
+
+def create_app(config_name):
```

Comment:
What does this function do?

Reply:
well that is the code that launches my app